about.html

```diff
@@ -0,0 +1,113 @@
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
+        </div>
+        <div class="hamburger">
+            <ul>
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+        </div>
+        <div  class="flex" >
+            <a  class="flex logo" href="index.html"><img  id="logo"  src="assets/Earth.png"/></a>
+            <ul class="flex nav-menu">
+              <li class="nav-item">
+              <a class="nav-link" id="nav1"  href="index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Program</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+            <li class="nav-item">
+              <a class="nav-link"  href="#">News</a>
+            </li>         
+            <li class="nav-item GE">
+                <a class="nav-link" id="ge" href="#">Campaign</a>
+              </li>    
+          </ul>
+          </div>
+    </nav>
+   <!--Header-->
+   <header>
+    <h1 class="heading1about">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
+    <p  class="paragabout">Human existence is quite impossible without the presence of a healthy ecosystem. Our environment comprises all living and non-living components and their interactions within a natural habitat. Environmental conservation has become one of the core issues that need to be addressed to battle climate change and global warming.Green Earth Global aims at conservation of natural resources. </p>
+      </header>
+
+<!-- more info -->
+  <section class="info-sec">
+    <p class="info-p">please contact us for further questions<br> regarding our annual summit<br>
+        <a class="info" href="#">info@GreenEarth.org</a>
+    </p>
+  </section>
+
+<!-- past summit -->
+<section class="past-summit">
+    <h3 class="summit">Green Earth Global Summit 2020</h3>
+    <hr class="hrabout">
+    <p class="majisafiP">The logo of Green Earth organisation was changed as of 21st July, 2020. We thank everyone who took part in voting for the logo below.</p>
+    <img class="majisafi" src="assets/Earth.png" alt="logo"/>
+</section>
+
+<!-- Past pictures -->
+<section class="pastphotos">
+    <h4 class="pastp">See the past G.E Global summit </h4>
+    <hr class="hrabout">
+    <p class="pag"> Take a look at the past two G.E global summit which took place<br>in Buenos Aires and Warsaw.</p>
+    <div >
+        <ul class="group-photo">
+        <li >
+             <img class="grp" src="assets/group1.png" alt="group photo"/>
+                </li>
+        <li ><img class="grp" src="assets/group2.png" alt="group photo"/>
+            </li>
+    </ul>
+    </div>
```

Comment:
- [ ] Your past event images are too small on the mobile versions. Please just change their alignment but keep their sizes almost the same.

**Design**
![img1](https://github.com/fatmahussein/Green-Earth-Summit/assets/99656687/b6e741bb-86c7-40a8-b940-3f7db35376ee)

**Your project**
![img2](https://github.com/fatmahussein/Green-Earth-Summit/assets/99656687/e3e73db2-71fc-4c83-9620-4cc8d3878140)